bin/node/runtime/src/lib.rs

```diff
@@ -521,14 +523,25 @@ parameter_types! {
 		.get(DispatchClass::Normal)
 		.max_extrinsic.expect("Normal extrinsics have a weight limit configured; qed")
 		.saturating_sub(BlockExecutionWeight::get());
+	// Solution can occupy 90% of normal block size
+	pub MinerMaxLength: u32 = Perbill::from_rational(9u32, 10) *
+		*RuntimeBlockLength::get()
+		.max
+		.get(DispatchClass::Normal);
```

Comment:
Makes it possible to an offchain worker always submit "something acceptable" to the chain. 
[The issue](https://github.com/paritytech/substrate/issues/8315)
[Original PR](https://github.com/paritytech/substrate/pull/8317)